app/helpers/class-user-table.js

```diff
@@ -0,0 +1,15 @@
+import { helper } from '@ember/component/helper';
+
+export function classUserTable([userType]) {
+  let newTableClass = 'table-primary';
+  if (userType === 'requestor') {
+    newTableClass = 'table-success';
+  } else if (userType === 'reqponder') {
+    newTableClass = 'table-warning';
+  } else if (userType === 'admin') {
+    newTableClass = 'table-danger';
+  }
+  return newTableClass;
+}
+
+export default helper(classUserTable);
```

Comment:
Small tip for this condition case, what do you think about that?:
```suggestion
import { helper } from '@ember/component/helper';

const tableClass = {
  requestor: 'table-success',
  responder: 'table-warning',
  admin: 'table-danger'
}

export function classUserTable([userType]) {
  return tableClass[userType] || 'table-primary';
}

export default helper(classUserTable);
```